logging/telemetryspec/metric.go

```diff
@@ -129,3 +136,49 @@ type RoundTimingMetrics struct {
 func (m RoundTimingMetrics) Identifier() Metric {
 	return roundTimingMetricsIdentifier
 }
+
+type transactionProcessingTimeDistibution struct {
+	// 10 buckets: 0-100Kns, 100Kns-200Kns .. 900Kns-1ms
```

Comment:
I slightly wish this was uniform log scale buckets. To keep things int it could be power of two buckets? <= 1ns, <= 2ns, <= 4ns, etc. It would make AddTransaction simpler. Not at all a big deal.

Reply:
I baffled with that as well. I think that this would become more valuable as we try to optimize non-trivial transactions. 
Having a "pure" logarithmic scale would place too many "entries" in a single bucket and would hurt the resolution we'd like to have.
To be concrete : for the block assembly, we really want *all* the transactions to be on the lower bucket. Any time this is not the case, we need to try and figure out why.